src/backend/api/library.ts

```diff
@@ -66,7 +66,7 @@ export const handleLaunchGame = (
     event: Electron.IpcRendererEvent,
     appName: string,
     runner: Runner
-  ) => Promise<{ status: 'done' | 'error' }>
+  ) => Promise<{ status: 'done' | 'error' | 'abort' }>
```

Comment:
this was required after making a change to make the update and install commands more consistent: we were able to abort updates but the status was `done` instead of `abort` which doesn't match the installation status that was `abort` when aborted. That change had some ramifications on the types of things